### ISSUE TITLE
How can I add to the functionality of every row field?

### ISSUE THREAD

**Issue by NewBo1234** [tabalinas/jsgrid#1089]
I have a jsGrid (awesome control by the way) that I would like to add the ability for the user to press enter in any field and then it will update/insert the record.  About half the fields are standard fields ("text", "number" or "select") and about half are custom fields.

I don't want to interrupt the current flow--the changing of the background, the substituting of the edit controls, etc--just to be able to loop through the controls on the editing row (or the inserting row) and touch each control to add an onKeyDown event that will trigger the established updateItem function.  

Basically, it is a working jsGrid.  I want to add the ability to hit the enter key and keep going.
